trace-test-application/src/androidTest/java/io/bitrise/trace/testapp/BaseUiTest.java

```diff
@@ -62,14 +66,15 @@
     protected static final int WAIT_FOR_IDLE_TIMEOUT = 10000;
     protected static final int SCREENSHOT_COMPRESSION = 100;
     protected static UiDevice uiDevice;
-    protected static final String UI_TEST_TAG = "Trace UI Test";
+    public static final String UI_TEST_TAG = "Trace UI Test";
     private static final int LAUNCH_TIMEOUT = 15000;
+    private static final String anrText = "isn't responding";
 
     /**
-     * Rule for taking a screenshot every time a test case fails.
+     * Rule for collecting data for every test case.
```

Comment:
nit: I'm not sure this is true for every test case - i don't think the espresso tests use this?

Reply:
Screenshots and view hierarchy xmls are created for every subclass of it, including the UrlConnectiionUiTest class.

Reply:
Yeah, i think the suggestion here is to remove "for every test case" to maybe "for test cases that inherit from this class"?

Reply:
Sure, I can add it.